docs/src/user-guide/integrations.md

```diff
@@ -8,6 +8,10 @@ eleventyNavigation:
 
 ---
 
+This page contains community projects that have integrated ESLint. The projects on this page are not maintained by the ESLint core team.
```

Comment:
We don't typically use the terminology "core team", so just "ESLint team" should do.

```suggestion
This page contains community projects that have integrated ESLint. The projects on this page are not maintained by the ESLint team.
```